### DISCUSSION TITLE
An Option to make the Emote Picker switch to the current Channel of the Window it was opened from

### DISCUSSION THREAD

**Discussion by Adleran12** [Chatterino/chatterino2#4283] [Ideas]
In the Current Version i need to open a new Emote-picker every time i switch between Chats because it posts in the one i opened it first, which is a hassle and also filling up my screen-space very fast if i don't close the old ones.

 so an Option to make the Emote-picker in total or just a Specific Window change to the newly selected Chat of the Window i opened it from or have it switch when i click on the Text bar would be very Helpful. 

with friendly regards,
That Guy who reads in far too many Chats far too often.

**Comment by leon-richardt** [comment]
Would there be any reasons against always switching the emote picker to the currently focused split (i.e., it wouldn't be an option but default behavior)?

**Reply by Adleran12** [reply]
that would also work, i just didn't want to say that they should change a Default because i don't think i'm a very Default user.
So in my Eyes at least the would be no reason against it.
